lib/language_pack/ruby.rb

```diff
@@ -456,11 +464,21 @@ def install_binaries
   # @param [String] name of the binary package from S3.
   #   Example: https://s3.amazonaws.com/language-pack-ruby/node-0.4.7.tgz, where name is "node-0.4.7"
   def install_binary(name)
+    topic "Installing #{name}"
     bin_dir = "bin"
     FileUtils.mkdir_p bin_dir
     Dir.chdir(bin_dir) do |dir|
       if name.match(/^node\-/)
         @node_installer.install
+        node_bin_path = File.absolute_path(".")
+        ENV["PATH"] = "#{ENV["PATH"]}:#{node_bin_path}"
```

Comment:
we need this b/c we need the absolute path during `yarn install`. some node modules using `node-gyp` will change the CWD and try to execute `node` and it won't be there.